pynteny/filter.py

```diff
@@ -401,14 +396,17 @@ def add_HMM_meta_info_to_hits(self, hmm_meta: Path) -> SyntenyHits:
             return self._synteny_hits
         pgap = PGAP(hmm_meta)
         self._synteny_hits[fields] = ""
-        for i, row in self._synteny_hits.iterrows():
+        # for i, row in self._synteny_hits.iterrows():
+        for row in self._synteny_hits.itertuples():
+            i = getattr(row, "Index")
+            hmm_group = getattr(row, "hmm")
```

Comment:
This can be changed to 
```py
row.Index
row.hmm
```

Not so ugly anymore, right? 😸

Reply:
I see! that pleases me greatly 😄  I really did not want to use `getattr`.